network/p2p/p2p.go

```diff
@@ -320,35 +327,6 @@ func netAddressToListenAddress(netAddress string) (string, error) {
 	return fmt.Sprintf("/ip4/%s/tcp/%s", ip, parts[1]), nil
 }
 
-// GetPeerTelemetryInfo returns the telemetry ID of a peer by looking at its protocols
-func GetPeerTelemetryInfo(peerProtocols []protocol.ID) (telemetryID string, telemetryInstance string) {
```

Comment:
removed b/c protocols fetch in `Connected` notification handler is racy

Reply:
Asked elsewhere what would happen if a node version prior to when this is released attempts to connect to an upgraded node - will it have any issues with the telemetryID being formatted differently?

Reply:
currently most likely our node will not get another's telemetryID. i.e. there is no loss of what does not really work.